deepmd/pd/utils/stat.py

```
@@ -0,0 +1,604 @@
+            system["coord"],
+            system["atype"],
+            system["box"],
+            system["natoms"],
+        )
+        fparam = system.get("fparam", None)
+        aparam = system.get("aparam", None)
+
+        def model_forward_auto_batch_size(*args, **kwargs):
+            return auto_batch_size.execute_all(
+                model_forward,
+                nframes,
```

Ruff

Function definition does not bind loop variable `nframes`

(B023)